### PR TITLE
Script - Deploy IPv6 dual stack web application in Azure virtual network

### PR DESCRIPTION
**Pull request by KumudD**

### CODE REVIEW

**Comment by KumudD** [comment]
#sign-off

File: virtual-network/ipv6-vnet-with-two-vms/ipv6-vnet-with-two-vms.ps1 [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+-SecurityRules $rule1,$rule2
+
+#Create Virtual Network and Subnet
+# Create dual stack subnet config
+$subnet = New-AzVirtualNetworkSubnetConfig `
+-Name "dsSubnet" `
+-AddressPrefix "10.0.0.0/24","ace:cab:deca:deed::/64"
+
+# Create the virtual network
+$vnet = New-AzVirtualNetwork `
```
Comment (sptramer) [inline virtual-network/ipv6-vnet-with-two-vms/ipv6-vnet-with-two-vms.ps1]: Got an error here:
```
New-AzVirtualNetwork : Subscription .... is not registered for feature Microsoft.Network/AllowIPv6VirtualNetwork required to 
carry out the requested operation.
```

If there are additional configuration flags that need to be set to allow IPv6, they should either be called out as prerequisites in the comments or toggled by the script itself.

File: virtual-network/ipv6-vnet-with-two-vms/ipv6-vnet-with-two-vms.ps1 [hunk 2]
```diff
@@ -0,0 +1,154 @@
+﻿# Dual-Stack VNET with 2 VMs.ps1
+#	Deploys Dual-stack (IPv4+IPv6) VNET with 2 VM's and BASIC Load Balancer with IPv4 and IPv6 Public IP's
+
+#Create Resource Group to contain the deployment
+$rg = New-AzResourceGroup `
+-ResourceGroupName "dsRG1"  `
```
[the remark this answers is not in the document]
Comment (KumudD) [reply virtual-network/ipv6-vnet-with-two-vms/ipv6-vnet-with-two-vms.ps1]: I have removed all line continuations as suggested. Thanks!

File: virtual-network/ipv6-vnet-with-two-vms/ipv6-vnet-with-two-vms.ps1 (hunk 1 above)
Comment (KumudD) [reply virtual-network/ipv6-vnet-with-two-vms/ipv6-vnet-with-two-vms.ps1]: I have added the registration information at the beginning of the script. The registration is to be only done once. Thanks!

**Comment by KumudD** [comment]
#sign-off

**Comment by sptramer** [comment]
This is great, thank you!

**Comment by KumudD** [comment]
@sptramer Thank you!

**Pull request merged**